classes/Domain/Services/ResetEmailer.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace OpenCFP\Domain\Services;
+
+class ResetEmailer
```

Comment:
Let's just leave it as-is for now. I'm fine to merge this otherwise, though I want to give it a little bit deeper look when I get home. More tests and tests passing is always a good thing.